build/build.js

```diff
@@ -22,8 +23,16 @@ const get = async (url, binary = true) => {
   return binary ? res.buffer() : res.text();
 };
 
+const force = process.argv.slice(2).some((arg) => ['--force', '-f'].includes(arg));
```

Comment:
Is it ok to pass args to build like this?

Reply:
this is fine by me, but we should include the env check in the same line, so
```suggestion
const force = process.argv.slice(2).some((arg) => ['--force', '-f'].includes(arg)) || process.env.FORCE === 'true';
```

that is probably too long, so might need to lint locally